lib/ea_restaurant_data_loader_web/router.ex

```diff
@@ -1,12 +1,27 @@
 defmodule EaRestaurantDataLoaderWeb.Router do
   use EaRestaurantDataLoaderWeb, :router
+  use Plug.ErrorHandler
+  alias EaRestaurantDataLoader.Lib.ErrorHandlers.InvalidCredentialsError
 
   pipeline :api do
-    plug :accepts, ["json"]
+    plug(:accepts, ["json"])
   end
 
-  scope "/api", EaRestaurantDataLoaderWeb do
-    pipe_through :api
+  scope "/", EaRestaurantDataLoaderWeb.Controllers do
+    pipe_through([:api])
+
+    post("/login", Oauth2Controller, :login)
+  end
+
+  @impl Plug.ErrorHandler
+  def handle_errors(conn, %{kind: _kind, reason: reason, stack: _stack}) do
```

Comment:
Can you create your own file to handle errors?

Reply:
Done.